src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1308,7 +1312,7 @@ public boolean verifyServerHostKey(String hostname, int port, String serverHostK
 
                         return getSshHostKeyVerificationStrategyDefaulted().verify(computer, key, listener);
                     }
-                });
+                }, launchTimeoutMillis, 0 /*read timeout - JENKINS-48618*/, launchTimeoutMillis);
```

Comment:
Does this behave as before when `launchTimeoutMillis == 0`?

Reply:
Yes, that's what the original method passes to this one